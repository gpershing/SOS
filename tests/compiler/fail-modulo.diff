diff: ./tests/compiler/tests/compiler/fail-modulo.err: No such file or directory
