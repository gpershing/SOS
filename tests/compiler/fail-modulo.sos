float a = 1.5

3^a
