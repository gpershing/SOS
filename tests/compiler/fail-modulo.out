Cannot exponentiate to a non-integer power
